pkg/resource/deploy/deployment_executor.go

```diff
@@ -236,7 +236,15 @@ func (ex *deploymentExecutor) Execute(callerCtx context.Context, opts Options, p
 				}
 
 				if event.Event == nil {
-					return false, ex.performDeletes(ctx, updateTargetsOpt, destroyTargetsOpt)
+					res := ex.performDeletes(ctx, updateTargetsOpt, destroyTargetsOpt)
```

Comment:
I think I added this while debugging some unplanned delete bugs. I think it's safe to stay here, means you get a nicer error if deletes do fail and it just translated into a Bail later on anyway.